docs/flow/reference/components/prometheus.remote_write.md

```diff
@@ -0,0 +1,357 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/prometheus.remote_write
+title: prometheus.remote_write
+---
+
+# prometheus.remote_write
+
+`prometheus.remote_write` collects metrics sent from other components into a
+Write-Ahead Log (WAL) and forwards them over the network to a series of
+user-supplied endpoints.
+
+Multiple `prometheus.remote_write` components can be specified by giving them
+different labels.
+
+## Example
+
+```river
+prometheus.remote_write "staging" {
+  // Send metrics to a locally running Mimir.
+  endpoint {
+    url = "http://mimir:9009/api/v1/push"
+  }
+}
+
+// Configure a prometheus.scrape component to send metrics to
+// prometheus.remote_write component.
+prometheus.scrape "demo" {
+  targets = [
+    {"__address__" = "demo.robustperception.io:9090"},
```

Comment:
I wonder about using robustperception in our docs since it is not "owned" by the Agent.

Reply:
Yeah, that's fair, I really wish Grafana had a demo server like this. I'll change it out to `localhost:12345`